packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -1856,4 +1857,20 @@ describe('RpcContractProvider test', () => {
       done();
     });
   });
+
+  describe('Testing Bug #1762 mapTypecheckError', () => {
+    it('should have a defined response and storage of 3 should not throw an error', async (done) => {
```

Comment:
you can put the issue # in the it block like so:
```
it('should return storage without errors (#1762)', ...
```

or even a comment above the `it` block linking to the issue itself